src/main/kotlin/com/demonwav/mcdev/i18n/I18nEditorNotificationProvider.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n
+
+import com.demonwav.mcdev.i18n.lang.I18nFileType
+import com.demonwav.mcdev.i18n.lang.gen.psi.I18nTypes
+import com.demonwav.mcdev.util.runWriteAction
+import com.intellij.openapi.editor.colors.EditorColors
+import com.intellij.openapi.editor.colors.EditorColorsManager
+import com.intellij.openapi.fileEditor.FileEditor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.Key
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.PsiManager
+import com.intellij.ui.EditorNotificationPanel
+import com.intellij.ui.EditorNotifications
+import com.intellij.util.ui.UIUtil
+import java.awt.Color
+
+class I18nEditorNotificationProvider(private val project: Project) : EditorNotifications.Provider<I18nEditorNotificationProvider.InfoPanel>() {
+    private var show: Boolean = true
+
+    override fun getKey() = KEY
+
+    override fun createNotificationPanel(file: VirtualFile, fileEditor: FileEditor): InfoPanel? {
+        if (!show || file.fileType != I18nFileType || file.nameWithoutExtension.toLowerCase() == I18nConstants.DEFAULT_LOCALE) {
```

Comment:
You need to be careful when using `toLowerCase()` without a locale (e.g. `Locale.ROOT` if you don't want to handle any locale specific cases), because with some locales (e.g. Turkey) some characters get translated to entirely different ones.